docker/Dockerfile

```diff
@@ -17,8 +17,11 @@ RUN apt-get update && apt-get install -y wget
 RUN apt-get update && apt-get install -y lsof dnsutils netcat-openbsd net-tools curl jq # used by integration tests
 
 # ruby and jazzy for docs generation
-RUN apt-get update && apt-get install -y ruby ruby-dev libsqlite3-dev
-RUN gem install jazzy --no-ri --no-rdoc
+RUN apt-get update && apt-get install -y ruby ruby-dev libsqlite3-dev build-essential
+
+# jazzy no longer works on xenial as ruby is too old.
+RUN if [ "${ubuntu_version}" = "focal" ] ; then echo "gem: --no-document" > ~/.gemrc; fi
+RUN if [ "${ubuntu_version}" = "focal" ] ; then gem install jazzy; fi
```

Comment:
Better read `$VERSION_ID` (eg. `20.04`) from `/etc/os-release` instead of directly comparing the code name.

Reply:
I'll try that, thanks